benchy

```diff
@@ -264,6 +264,7 @@ print_virtualization() {
     echo "openvz" && return 0
   fi
   # wsl test
+  if ! [ -d "/proc/version" ]; then return 0; fi
```

Comment:
Did you mean `if ! [ -f "/proc/version" ]; then return 0; fi` ? To my knowledge `/proc/version` is a file containing version information and not a directory-- but you test for a directory (`-d`).